src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ApprenticeshipIncentive.cs

```diff
@@ -186,6 +186,19 @@ public void ValidateHasNoDataLocks(Guid pendingPaymentId, Learner matchedLearner
             pendingPayment.AddValidationResult(PendingPaymentValidationResult.New(Guid.NewGuid(), collectionPeriod, ValidationStep.HasNoDataLocks, !hasDataLock));
         }
 
+        public void ValidateDaysInLearning(Guid pendingPaymentId, Learner matchedLearner, CollectionPeriod collectionPeriod)
+        {
+            var pendingPayment = GetPendingPaymentForValidationCheck(pendingPaymentId);
+
+            var hasEnoughDaysInLearning = false;
+            if (matchedLearner != null)
+            {
+                hasEnoughDaysInLearning = matchedLearner.GetDaysInLearning(collectionPeriod) >= 90;
```

Comment:
What happens with the 2nd payment?  We should be using the rules from the incentive object.

Reply:
Done as discussed.